modules/shell/command.go

```diff
@@ -85,12 +86,20 @@ func RunCommandAndGetOutputE(t *testing.T, command Command) (string, error) {
 }
 
 // This function captures stdout and stderr while still printing it to the stdout and stderr of this Go program
-func readStdoutAndStderr(t *testing.T, stdout io.ReadCloser, stderr io.ReadCloser) (string, error) {
+func readStdoutAndStderr(t *testing.T, stdout io.ReadCloser, stderr io.ReadCloser, maxLineSize *int) (string, error) {
 	allOutput := []string{}
 
 	stdoutScanner := bufio.NewScanner(stdout)
 	stderrScanner := bufio.NewScanner(stderr)
 
+	if maxLineSize != nil {
+		stdoutBuf := make([]byte, maxLineSize)
+		stderrBuf := make([]byte, maxLineSize)
```

Comment:
couldn't we just put this inline to the Buffer creation below, and save a few lines?

```go
if maxLineSize != nil {
    stdoutScanner.Buffer(make([]byte, maxLineSize), maxLineSize)
    stderrScanner.Buffer(make([]byte, maxLineSize), maxLineSize)
}
```

Reply:
done